x/stake/test_common.go

```diff
@@ -123,6 +149,13 @@ func createTestInput(t *testing.T, sender sdk.Address, isCheckTx bool, initCoins
 	)
 	ck := bank.NewCoinKeeper(accountMapper)
 	keeper := NewKeeper(ctx, cdc, keyStake, ck)
+	encoded, err := json.Marshal(GenesisState{initialPool(), defaultParams()})
+	if err != nil {
+		panic(err)
+	}
+	if err = keeper.InitGenesis(ctx, encoded); err != nil {
+		panic(err)
+	}
```

Comment:
Okay, I notice here that the InitGenesis functionality is almost "tested" here... let's move this to a separate explicit test for InitGenesis (aka `TestInitGenesis`) in `keeper_test.go` and replace lines 152-158 with:
``` golang
k.setPool(ctx, state.Pool)
k.setParams(ctx, state.Params)
```
The function createTestInput is not really supposed to a test itself just a helper for other actual tests hence it's not in a `_test.go` file

Reply:
Updated, moved to `TestInitGenesis`.